build.gradle

```diff
@@ -180,19 +175,17 @@ task pushDockerImage {
 
 task removeOldDockerContainer(type: Exec) {
     commandLine "bash", "-c",
-            "for DC in `docker ps -f 'name=$dockerContainerName' | grep -o $dockerContainerName`; do docker stop \$DC; done && " +
-            "for DC in `docker ps -a -f 'name=$dockerContainerName' | grep -o $dockerContainerName`; do docker rm \$DC; done"
+            "docker ps -aqf name=$dockerContainerName | xargs -r docker rm -f"
```

Comment:
Unfortunately, not, since `-n 1` is "up to 1" argument. Means that it will still treat the empty line as one argument and `docker -rm` complains about a missing parameter. On a Mac it works even without `-n 1` because the BSD xargs ignores the empty line.